### ISSUE TITLE
Is this repo still alive.

### ISSUE THREAD

**Issue by alanrogers** [BioJulia/PopGen.jl_archive#5]
Just found this repo, but I notice that there are only two commits, the last of which was in 2015. Is this project still alive?

**Comment by alejandromerchan** [comment]
There's development happening in another [fork](https://github.com/pdimens/PopGen.jl). I believe the idea is to eventually move the functionality back into this repo, but I'm not sure.

**Comment by pdimens** [comment]
@alanrogers sorry, I finally saw this. Yes, there is active development on my fork, and @BenJWard and I need to have a conversation about either merging my fork with the BioJulia repo, or killing off the latter.

**Comment by alanrogers** [comment]
I'm a population geneticist and just starting with Julia. But I hope to
contribute to this repo if possible. Please keep me in the loop.

**Comment by pdimens** [comment]
@alanrogers oh absolutely! Many hands make light work :)
Have a look at the current state of things over at pdimens/popgen.jl and let me know what you think.

**Comment by blahah** [comment]
@pdimens for discoverability it would be ideal if your fork can be merged into BioJuila and you become maintainer. @BenJWard can we make that happen if @pdimens is keen?

**Comment by pdimens** [comment]
@blahah @BenJWard I'm definitely keen on it

**Comment by TransGirlCodes** [comment]
Heya @pdimens, I've sent an invite to BioJulia for you. If you'd like to accept and then perhaps make a PR to move the relevant branc(es) over. If you're happy to be listed on biojulia.net (no pressure, it's optional), either send me an email with a short bio I'll update biojulia.net, or you can make a PR to the website repo yourself.

**Comment by pdimens** [comment]
@BenJWard Sure, I can do that. Does being part of the org give me the same read/write access to the repo?

**Comment by TransGirlCodes** [comment]
@pdimens Yes, you'll have read/write access, and if you need more I can set you up!

We'll also make sure the repo has a code-owners file with you listed.

**Comment by pdimens** [comment]
Thanks. It says I don't have rights to publish/create new branches.

**Comment by TransGirlCodes** [comment]
@pdimens Try now, I should have added you to the group with write/push with the org invitation - I must have missed it. Should be all good now though.

**Comment by TransGirlCodes** [comment]
Ok, the process of merging PopGen.jl is underway and will be finished soon. To go back to @alanrogers first post, the repo is still alive, are you happy for the issue to be closed/resolved?

**Comment by kescobo** [comment]
:zombie:

**Comment by pdimens** [comment]
@alanrogers my fork of PopGen.jl has been fully merged back into BioJulia/PopGen.jl and future development and releases will happen from this repo, which is now considered the official upstream. In addition, I'm also one of the admin for this repo.

With your consent, may we close this issue?
